### PR TITLE
Use new ApplicationsService.getApplications utility

### PR DESCRIPTION
**Pull request by jeff-phillips-18**

### CODE REVIEW

**Comment by jeff-phillips-18** [comment]
Removed WIP status

**Comment by spadgett** [comment]
[merge][severity: bug]

File: app/scripts/directives/addSecretToApplication.js [hunk 1]
```diff
@@ -21,63 +22,16 @@
...
-        sortApplications();
-      });
-      DataService.list({
-        group: 'apps',
-        resource: 'statefulsets'
-      }, context).then(function(statefulSetData) {
-        statefulSets = _.toArray(statefulSetData.by('metadata.name'));
-        sortApplications();
+      ApplicationsService.getApplications(context).then(function(applications) {
+        ctrl.applications = _.sortBy(applications, ['metadata.name', 'kind']);
```
Comment (spadgett) [inline app/scripts/directives/addSecretToApplication.js]: ApplicationService sorts, so we shouldn't need to sort here.

https://github.com/openshift/origin-web-common/blob/master/src/services/applicationsService.js#L28

File: app/scripts/directives/bindService.js [hunk 2]
```diff
@@ -106,41 +94,13 @@
...
-        sortApplications();
-      });
-      DataService.list({
-        group: 'apps',
-        resource: 'statefulsets'
-      }, context).then(function(statefulSetData) {
-        statefulSets = _.toArray(statefulSetData.by('metadata.name'));
-        sortApplications();
+      ApplicationsService.getApplications(context).then(function(applications) {
+        ctrl.applications = _.sortBy(applications, ['metadata.name', 'kind']);
```
Comment (spadgett) [inline app/scripts/directives/bindService.js]: Shouldn't need to sort here.

File: app/scripts/directives/addSecretToApplication.js (hunk 1 above)
Comment (jeff-phillips-18) [reply app/scripts/directives/addSecretToApplication.js]: Fixed

File: app/scripts/directives/bindService.js (hunk 2 above)
Comment (jeff-phillips-18) [reply app/scripts/directives/bindService.js]: Fixed

**Comment by spadgett** [comment]
[merge]

**Pull request merged**